src/Parallelization.php

```diff
@@ -389,6 +389,7 @@ protected function executeMasterProcess(InputInterface $input, OutputInterface $
                     '--child',
                     '--env='.$input->getOption('env'),
                     $input->getOption('no-debug') ? '--no-debug' : '',
+                    implode(" ", $this->serializeInputOptions($input, ['child','env','no-debug', 'processes'])),
```

Comment:
I think a better option, rather than blacklisting `env` and `no-debug`, is to leave them but removing them L390 and L391

Reply:
Yes, that was also in my mind.
I will change that.